oauth/oauth.go

```diff
@@ -27,15 +27,20 @@ type Config struct {
 	RefreshToken string
 }
 
-// NewConfig creates a new *Config for the provided client credentials.
+// NewConfig creates a new *Config for the provided client credentials and default token URL.
 func NewConfig(clientID, clientSecret, refreshToken string) *Config {
+	return NewConfigWithBaseUrl(clientID, clientSecret, refreshToken, TokenURL)
+}
+
+// NewConfig creates a new *Config for the provided client credentials and token URL.
+func NewConfigWithBaseUrl(clientID, clientSecret, refreshToken, tokenUrl string) *Config {
```

Comment:
I just noticed that this is not really used anywhere so it could also just be removed as https://github.com/Bonial-International-GmbH/site24x7-go/pull/34/files#diff-4b667feae66c9d46b21b9ecc19e8958cf4472d162ce0a47ac3e8386af8bbd8cfR49-R52 already accounts for the custom `tokenURL`.

Reply:
True, I removed all changes to oauth.go.